java/client/build.gradle

```diff
@@ -96,23 +97,12 @@ tasks.register('buildRustReleaseStrip', Exec) {
     environment CARGO_TERM_COLOR: 'always'
 }
 
-tasks.register('buildRust', Exec) {
-    commandLine 'cargo', 'build'
-    workingDir project.rootDir
-    environment CARGO_TERM_COLOR: 'always'
-}
-
 tasks.register('buildRustFfi', Exec) {
     commandLine 'cargo', 'build'
     workingDir project.rootDir
     environment CARGO_TERM_COLOR: 'always', CARGO_BUILD_RUSTFLAGS: '--cfg ffi_test'
 }
 
-tasks.register('buildWithRust') {
-    dependsOn 'buildRust'
-    finalizedBy 'build'
-}
```

Comment:
we still need to keep non-release build for development, why is it being removed?

Reply:
We haven't actually been using the debug build for development. I can keep it though if you really want.